sdk/core/azure-core/README.md

```diff
@@ -19,9 +19,40 @@ APIs in one client library, you will know how to use them in other client librar
 
 ### Include the package
 
-Typically, you won't need to install or depend on Azure Core, instead it will be transitively downloaded by your build
+#### Include the BOM file
+
+Please include the azure-sdk-bom to your project to take dependency on GA version of the library. In the following snippet, replace the {bom_version_to_target} placeholder with the version number.
```

Comment:
> Please include the azure-sdk-bom to your project to take dependency on GA version of the library.

Please include the azure-sdk-bom to your project to take **a** dependency on **the General Availability (GA)** version of the library.